app/javascript/components/enrollment/steps/Review.js

```diff
@@ -66,18 +66,14 @@ class Review extends React.Component {
           <Card.Body>
             <Patient
               goto={this.props.goto}
-              edit_mode={true}
+              hidePreviousButton={true}
```

Comment:
I wouldn't change this prop name because the Patient component has no knowledge of a previous button and it doesn't really make any sense

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/6620f09e6346a83c99080db3e9e12eef8a1ee5f4